cypress/constants/tests.ts

```diff
@@ -8,3 +8,21 @@ export const STATUS_CARD_TITLE = 'Data Federation service';
 export const SINGLE_BUCKET_POLICY = 'e2e-bucket-single';
 export const PVC_NAME = 'e2e-pvc';
 export const DATA_SOURCE_NAME = 'e2e-nsfs-data-source';
+
+// constants for data source test cases
+export const TEST_DATA_SOURCE = 'testing-data-source';
+export enum Providers {
+  AWS = 'AWS S3',
+  AZURE = 'Azure Blob',
+  S3 = 'S3 Compatible',
+  PVC = 'PVC',
+  IBM = 'IBM COS',
+  FILESYSTEM = 'Filesystem',
+}
+export const dataSourceCreateInputs = {
```

Comment:
For naming consistency, provided that `export const TEST_DATA_SOURCE` is in capital+snake case, can you do the same here?